### DISCUSSION TITLE
Make a way to display whats playing to discord

### DISCUSSION THREAD

**Discussion by janechicken** [navidrome/navidrome#3242] [Ideas]
The same way spotify does it that no matter what device you do it on, I think the same could be done for navidrome and implement it on the server so it can do it like spotify instead of the discord rpc being done on the device

**Comment by deluan** [comment]
This is kind of low priority to implement in Navidrome itself, but there are some third-party solutions to add this feature. This one for example: https://github.com/MeapVR/Navicord

**Reply by Metal-Snake** [reply]
that link is dead now. But I found this https://github.com/logixism/navicord this works for me.
